configure.ac

```diff
@@ -1851,8 +1851,8 @@ AS_IF([test x"$DEFAULT_STRING" = "xunsafe"],
 
 oc_cflags="$common_cflags $internal_cflags"
 oc_cppflags="$common_cppflags $internal_cppflags"
-ocamlc_cflags="$common_cflags $sharedlib_cflags"
-ocamlc_cppflags="$common_cppflags"
+ocamlc_cflags="$common_cflags $sharedlib_cflags \$(CFLAGS)"
```

Comment:
Is this definitely what we want? It seems unquestionably correct that the build of the system respects `CFLAGS` given at build time and defaults to the `CFLAGS` at `configure` for building OCaml, but it doesn’t seem like `ocamlc -c` should *have* to respect `CFLAGS`. If it does, shouldn’t it be in the same way - i.e. shouldn’t `ocamlc -c` always append the *current* value of `CFLAGS`, defaulting to the value of `CFLAGS` when `config.mlp` was built if it’s not presently defined?